index.html

```diff
@@ -77,5 +74,417 @@
             </a>
         </div>
     </header>
+
+    <main>
+
+        <!-- categories component -->
+        <div class="container-fluid bg-white shadow-sm p-0">
+            <div class="container-lg d-flex justify-content-evenly align-items-center p-2 gap-4  overflow-x-auto">
+                <a 
+                    href="#" 
+                    class="categories d-flex flex-column align-items-center justify-content-center text-black text-decoration-none"
+                    aria-label="Categorie Top offers"
```

Comment:
There is spelling mistake here. (Applicable to all categories).

```suggestion
                    aria-label="Category Top offers"
```

Reply:
Fixed it.

https://github.com/krushna-simform/flipkart-skeleton-ui/pull/5/commits/faf6dd4b899baea3853af3a614bf1e7cb3a99822